compiler-core/src/language_server/tests/hover.rs

```diff
@@ -65,6 +76,269 @@ fn(Int) -> Int
     );
 }
 
+#[test]
+fn hover_local_function() {
+    let code = "
+fn my_fn() {
+  Nil
+}
+
+fn main() {
+  my_fn
+}
+";
+
+    assert_eq!(
+        positioned_hover(code, Position::new(6, 3)),
+        Some(Hover {
```

Comment:
It would be good to use snapshot tests with `cargo-insta` here rather than typing out the returned value, to match the majority of tests in the codebase and to reduce the amount of work maintaining the tests.